packages/menu-bar/src/vaadin-menu-bar-mixin.js

```diff
@@ -145,6 +156,9 @@ export const MenuBarMixin = (superClass) =>
       container.addEventListener('click', this.__onButtonClick.bind(this));
       container.addEventListener('mouseover', (e) => this._onMouseOver(e));
 
+      this._addHoverListener(this._subMenu);
+      this._addSubMenuOpenListener(this._subMenu);
```

Comment:
All the tests pass with this line removed. Needs a test where opening a sub-menu doesn't cause the menu bar to auto-close.